### DISCUSSION TITLE
Quarto Powerpoint Issues

### DISCUSSION THREAD

**Discussion by sailanki1609** [quarto-dev/quarto-cli#11017] [Q&A]
Here's a template .pptx (dogs_template.pptx) and the output file that is generated (dogs.pptx). I have also pasted the code in this text file(Dogs code.docx).  

I'm struggling with the below issues. Any help is appreciated. 

[dogs_template.pptx](https://github.com/user-attachments/files/17307613/dogs_template.pptx)
[dogs.pptx](https://github.com/user-attachments/files/17307660/dogs.pptx)
[Dogs code.docx](https://github.com/user-attachments/files/17307740/Dogs.code.docx)

 
1. **### Unable to specify slide layout in Quarto when using a reference .pptx template**
I have created a template .pptx document(dogs_template.pptx) and made some changes to it directly through the Slide Master View in Powerpoint. I am referencing this document through the YAML section. The title slide is pulled directly from the template, but I'm trying to create a new slide called  and set it layout to a specific layout say "Section Header" layout. How do i do that? 

I have tried a couple of syntaxes but nothing seems to work. 
![Image](https://github.com/user-attachments/assets/a90e308a-8946-4531-b39b-e453942751a4)

Is it possible to specify which slide layout i would like to apply while creating slides through quarto? 

2. **### Slide layout changes unexpectedly when adding an image**

I'm trying to create a slide with two bullet points at the top, followed by an image below. However, when I add the image, the layout of the slide changes to the "Content with Caption" layout. In this layout, the image moves to the second column, while the title and text remain in the first column.

Here are the codes that i tried out. I also tried to adjust the size of the image to make it fit below the text, but that didn't work. 

1st code-
![Image](https://github.com/user-attachments/assets/d1abf2bd-da57-4c39-983c-22c9a6d6619e)

2nd code- I tried inserting a non breaking space in between, and directly included the figure.
![Image](https://github.com/user-attachments/assets/d5b55a6d-6003-407b-a2fa-a8379673814e)

It looks correct in the Visual mode, but when i generate the powerpoint the layout is changing.

3. **### Unable to add two images vertically on a single slide**

I tried to lay 2 images, one below the other, but it is not happening. I used the {layout-nrow=2} attribute along with some other methods, but none of them worked. It is either resulting in one image per slide or one image overlaying the other.

These are the codes that i tried out. 
![Image](https://github.com/user-attachments/assets/e513b524-500f-46d4-bb47-64710888d779)

Slide 6 doesn't have any images on it, whereas Slide 7 results in 2 slides with images on 2 separate slides. I also user knitr::include_graphics() but that didn't work either. 

4. **### Is there a direct way to apply slide-specific styling in the same way HTML/CSS works.** 

I tried this code, although the changes are visible in the Visual Mode, but not coming correctly in the Powerpoint output. 

![Image](https://github.com/user-attachments/assets/be8b2786-5339-4e30-adc0-f50a6d11af20)

![Image](https://github.com/user-attachments/assets/7d9a0466-f0aa-4b9b-bccc-2f937d9ea77c)

5. **### Adding custom-footers for each slide.** 
I want to add a different footer for each slide. Is there a way to do that in quarto?

**Comment by mcanouil** [comment]
Converting to discussion.
Please read the guidelines about asking for help and reporting issues.
We unfortunately cannot help you because screenshots cannot be used.

Also, please do read the documentation about making a template for PowerPoint (or Word): https://quarto.org/docs/presentations/powerpoint.html#slide-layouts

Note that this is Pandoc doing the job of writing to PowerPoint: https://pandoc.org/MANUAL.html
